src/features/internal/marketingAndCommunication/atoms/OfferCategoryChoices.tsx

```diff
@@ -38,12 +39,12 @@ export const OfferCategoryChoices = (props: Props) => {
   return (
     <BodyContainer>
       <StyledUl>
-        {Object.entries(availableCategories).map(([category, { facetFilter }]) => (
-          <Li key={category}>
+        {availableCategories.map((category) => (
+          <Li key={category.facetFilter}>
             <SelectionLabel
```

Comment:
Si tu extrait un composant pour lui passer en props category.facetFilter tu devrais pouvoir te débarrasser du codesmell de onPress={() => onPress(category.facetFilter)}

```suggestion
            <SelectionLabelExt category={category.facetFilter} labels={searchGroupLabelMapping} selection={selection} onPress={onPress} />
            
            
            le onPress de SelectionLabelExt devrait ressembler à 
            const onPress = () => props.onPress(props.category)
            
```

@dbenfouzari une idée ?

Reply:
@pierrecanthelou Non, `<SelectionLabel />` n'est pas au courant qu'il manipule une `category`. Et je préfère autant que ça reste comme ça, puisqu'il ne fait qu'afficher un `label`.

Par contre je n'aime pas créer une fonction anonyme à chaque render.
Seulement, pour éviter ça, il faudrait un composant intermédiaire et bon... Pour si peu c'est pas ouf.

Reply:
Vu que les codes smells ont disparu je serai plus partant pour laisser comme ça, histoire de ne pas avoir un tout petit composant.